docs/en/tutorial.md

```diff
@@ -341,7 +341,169 @@ Let's run some queries to analyze the 2M rows of data...
     │ 2015-07-01 00:41:48 │ 2015-07-01 00:44:45 │          6.3 │                 -94 │                  132 │ JFK          │ 2015 │   1 │    0 │
     │ 2015-07-01 01:06:18 │ 2015-07-01 01:14:43 │        11.76 │                  37 │                  132 │ JFK          │ 2015 │   1 │    1 │
     ```
+## 4. Create a Dictionary
 
+If you are new to ClickHouse, it is important to understand how ***dictionaries*** work. A dictionary is a mapping of key->value pairs that is stored in memory.
```

Comment:
Maybe somehow to cover that a key could be both simple (a UInt64) and complex (multiple values). A value is actually multiple attributes. But in general Ok